include/view.h

```diff
@@ -184,6 +184,12 @@ void view_snap_to_edge(struct view *view, const char *direction,
 	bool store_natural_geometry);
 void view_snap_to_region(struct view *view, struct region *region,
 	bool store_natural_geometry);
+
+void view_move_to_front(struct view *view);
+/* This variant does not call cursor_update_focus() */
+void view_move_to_front_no_focus(struct view *view);
```

Comment:
Thoughts about using `view_move_to_front(struct view *view, bool focus)` vs. the two function variant?

Reply:
My thought was that `view_move_to_front()` was the normal case and that the only purpose of the `_no_focus` variant was to prevent recursion into `cursor_update_focus()`. So I didn't want to add a second parameter to all of the "normal" call sites.

Another possibility is to put a recursion guard in `cursor_update_focus()`; then `view_move_to_front()` doesn't need to care.

Reply:
Personally I'd slightly prefer the bool variant but I understand the argument for having two functions instead.
I am fine with both variants.

> Another possibility is to put a recursion guard in `cursor_update_focus()`; then `view_move_to_front()` doesn't need to care.

I personally would prefer that one to the current solution.
It would also reduce the chances of introducing something similar in the future.